src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -83,7 +83,7 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ proj, enter
 
   const handleClickFavorite = async () => {
     try {
-      mutate(proj.wbsNum);
+      mutate({});
```

Comment:
Is this the right way to call mutate? No params passed because it's passed into the hook directly

```tsx
const { mutate } = useToggleProjectFavorite(proj.wbsNum);
mutate({});
```

Reply:
I think you can call it without passing in the empty object @ninest

Reply:
Oops I think I had the wrong types for useMutation

No need for the `any` at the end:`return useMutation<{ message: string }, Error, any>(`